src/patternfly/components/Page/examples/Page.md

```diff
@@ -174,6 +174,6 @@ This component provides the basic chrome for a page, including sidebar, header,
 | `.pf-m-dark-200` | `.pf-c-page__main-section` |  Modifies a main page section to have a dark theme and a dark transparent background. |
 | `.pf-m-dark-100` | `.pf-c-page__main-section` |  Modifies a main page section to have a dark theme and a darker transparent background. |
 | `.pf-m-no-padding` | `.pf-c-page__main-section` | Modifies a main page section to have no padding. |
-| `.pf-m-no-padding-mobile` | `.pf-c-page__main-section` | Modifies a main page section to have no padding on mobile. |
+| `.pf-m-no-padding-{-on-[breakpoint]}` | `.pf-c-page__main-section` | Modifies a main page section to have no padding at a given breakpoint. |
```

Comment:
I think you could consolidate `.pf-m-no-padding` and this entry to `.pf-m-no-padding{-on[breakpoint]}` to say it removes padding at an optional breakpoint.